frontend/src/modules/batcheditor/components/BatchEditor.js

```diff
@@ -0,0 +1,121 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from 'fluent-react';
+
+import './BatchEditor.css';
+
+
+type Props = {
+    count: number,
+};
+
+
+/**
+ * Renders batch editor, used for performing mass actions on translations.
+ */
+export default class BatchEditor extends React.Component<Props> {
+    render() {
+        const { count } = this.props;
+
+        return <div className="batch-editor">
+            <div className="topbar clearfix">
+                <div className="selecting fa fa-sync fa-spin"></div>
+                <Localized
+                    id="batcheditor-BatchEditor--header-selected-count"
+                    attrs={{ title: true }}
+                    glyph={
+                        <i className="fa fa-times fa-lg"></i>
+                    }
+                    stress={ <span className="stress" /> }
+                    $count={ count }
+                >
+                    <button
+                        className="selected-count"
+                        title="Quit Batch Editing (Esc)"
+                    >
+                        { '<glyph></glyph> <stress>{ $count }</stress> strings selected' }
+                    </button>
+                </Localized>
+                <Localized
+                    id="batcheditor-BatchEditor--header-select-all"
+                    attrs={{ title: true }}
+                    glyph={
+                        <i className="fa fa-check fa-lg"></i>
+                    }
+                >
+                    <button
+                        className="select-all"
+                        title="Select All Strings (Ctrl + Shift + A)"
+                    >
+                        { '<glyph></glyph> Select All' }
+                    </button>
+                </Localized>
+            </div>
+
+            <div className="main-content">
+                <div className="intro">
+                    <p>
+                        <Localized id="batcheditor-BatchEditor--warning-title">
+                            <span className="warning">Warning:</span>
+                        </Localized>
+                        { ' ' }
+                        <Localized id="batcheditor-BatchEditor--warning-content">
+                            <span className="content">These actions will be applied to all selected strings and cannot be undone.</span>
+                        </Localized>
```

Comment:
Why is this split into 2 strings? I think it would make more sense to have only one, and use some tag to mark the `Warning:` part, maybe like `<mark>` maybe? Otherwise this structure will likely create issues in RTL languages, or other things I can't think of.